docs/source/host/maintain-domain/view-logs.rst

```diff
@@ -1,15 +1,11 @@
-.. warning::
-    This document is outdated.
-    FIXME: mentions High Fidelity, paths outdated, missing path for Linux servers
-
 #######################
 View Domain Server Logs
 #######################
 
 Your domain server logs list the different events relevant to your High Fidelity domain. These can help you troubleshoot any issues with your domain and understand how the different parts of your domain communicate with each other.
 
-Server logs are stored directly on the server in the following directories:
+Server logs are stored directly on the server and can be accessed as follows:
 
-* **Windows**: %AppData%/Local/High Fidelity/Server Console/logs
-* **Mac**: ~/Library/Application Support/High Fidelity/Server Console/Logs
+* **Windows**: Logs can be found in the directory "%AppData%/Local/Vircadia/Server Console/logs"
+* **Linux**: Logs can be found by running the command "journalctl -u vircadia-domain-server@default.service"
```

Comment:
```suggestion
* **Linux**: Logs can be accessed by running the command "journalctl -u vircadia-domain-server@default.service"
```
It doesn't show you where the logs are actually stored, so I would use different wording.

Also keep in mind that this will only show domain-server logs and not assignment-client logs. The assignment-clients do almost all the work. See https://docs.vircadia.com/explore/get-started/architecture.html#architecture-overview